addons/point_of_sale/static/src/js/Chrome.js

```diff
@@ -312,6 +314,9 @@ odoo.define('point_of_sale.Chrome', function(require) {
         _toggleDebugWidget() {
             this.state.debugWidgetIsShown = !this.state.debugWidgetIsShown;
         }
+        _toggleMobileSearchBar(event) {
+            this.state.mobileSearchBarIsShown = event.detail.mobileSearchBarIsShown;
+        }
```

Comment:
if i'm not wrong toggle is "implicit"
is not better to use something like _setMobileSearchBar(event) and in the event set the value ?